server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RecipientRewriteTableIntegrationTest.java

```diff
@@ -222,4 +222,32 @@ public void rrtServiceShouldFollowForwardWhenSendingToAGroup() throws Exception
             .awaitMessage(awaitAtMostOneMinute);
         assertThat(imapMessageReader.readFirstMessage()).isNotNull();
     }
+
+    @Test
+    public void domainAliasMappingShouldNotCreateNonExistedUserWhenReRouting() throws Exception {
```

Comment:
Can we assert a bounce is generated to explain correctly to the sender that there is no such user?

Reply:
as I understand the process goes through `RecipientRewriteTable` mailet where the address rewriting happens before going into other mailets, so there is no error of  `no such user` happened